src/commands/createNewProject/dockerfileSteps/DockefileProjectLanguageStep.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, type IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
+import { ProjectLanguage } from "../../../constants";
+import { type IDockerfileProjectContext } from "./IDockerfileProjectContext";
+
+export class DockerfileProjectLanguageStep extends AzureWizardPromptStep<IDockerfileProjectContext> {
```

Comment:
You could probably just use the `NewProjectLanguageStep` and add some filtering in the case for `DockerfileProjectLanguageStep` instead of creating a new file that does largely the same thing.

Reply:
Actually update since I am not using the NewProjectLanguageStep I don't think I can use the `language: ProjectLanguage` since the data types don't match up.

Reply:
Yeah but you should be able to just change the type instead of using a string.

Reply:
Looking at this more, I think we should just speak offline about this.